### ISSUE TITLE
Return array values when context is Element[]

### ISSUE THREAD

**Issue by jason0x43** [theintern/leadfoot#138]
Ideally, `getVisibleText` in this instance should resolve to an array:

```
remote
  .findAllByCssSelector('.item')
  .getVisibleText()
  .then(value => {
    // should be string[], but is string
  })
```

**Comment by jason0x43** [comment]
This will likely require the use of TS 2.8 conditional types or a separate CommandMulti (or something) type that will be returned by the `findAll` methods.

If we went the CommandMulti route, The CommandMulti type would just be an extension of Command that returned array values for the relevant methods (like `getVisibleText`).

**Comment by anitba** [comment]
should be fixed by the pull request 170 (Fix typings for Command element find all methods)

**Comment by jason0x43** [comment]
This was fixed in 3cdba41
